gocumber_test.go

```diff
@@ -201,3 +201,19 @@ func TestExec_NoMatch(t *testing.T) {
 
 	assert.False(t, steps.Exec("some unknown step"))
 }
+
+func TestDocstrings_OutlineVariables(t *testing.T) {
+	steps := make(Definitions)
+	tt := new(testing.T)
+
+	var called bool
+	steps.Given("something is:", func(_ []string, step StepNode) {
+		called = true
+		assert.Equal(t, "minimally functional\n\n", step.PyString().String())
```

Comment:
Two newlines is incorrect. If one comes from the initial parsing<sup>1</sup>, I'm fine with leaving it. *Adding* any is definitely not good.

<sup>1</sup> Gherkin tests show the final newline being omitted, but that should be done over in go-gherkin, I think.
 \- https://github.com/cucumber/gherkin3/blob/master/testdata/good/docstrings.feature.ast#L12-L16
 \- https://github.com/cucumber/gherkin3/blob/master/testdata/good/docstrings.feature.ast.json#L38